cyences_app_for_splunk/default/data/ui/views/cs_device_inventory_table.xml

```diff
@@ -332,14 +332,22 @@
     | eval second_bit_in_mac_binary=mvindex(binary,1) 
     | where second_bit_in_mac_binary=0 
     | eval Type="Mac Address Matches", ips="" 
-    | table Type, mac_addresses, uuid, no_uuids, hostnames, ip_all, mac_address_all, product_uuids] 
+    | table Type, mac_addresses, uuid, no_uuids, hostnames_all, ip_all, mac_address_all, product_uuids] 
     [| stats values(*) as * by ips 
     | where isnotnull(ips) AND ips!="" AND ips!=" " 
     | eval no_uuids=mvcount(uuid) 
     | where no_uuids&gt;1 
     | eval Type="IP Matches", mac_addresses="" 
-    | table Type, ips, uuid, no_uuids, hostnames, ip_all, mac_address_all, product_uuids] 
-| table Type, ips, mac_addresses, uuid, no_uuids, hostnames, ip_all, mac_address_all, product_uuids
+    | table Type, ips, uuid, no_uuids, hostnames_all, ip_all, mac_address_all, product_uuids] 
+    [| stats values(*) as * by hostnames 
+    | where isnotnull(hostnames) AND hostnames!="" AND hostnames!=" " 
+    | removepostfixes postfix_macro_name="cs_device_inventory_hostname_postfixes" field_to_check="hostnames" 
```

Comment:
Do we need remove prefixes??

Reply:
For device inventory we only need postfix. For user inventory, we are using both.